docs/mine/README.md

```diff
@@ -10,6 +10,24 @@ In most blockchain protocols, "miners" are the participants on the network that
 
 Mining in Filecoin works quite differently however -- instead of contributing computational power, miners contribute storage capacity to be used in deals with clients looking to store data.
 
+# Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Types of miners](#types-of-miners)
+  - [Coming from another ecosystem?](#coming-from-another-ecosystem)
+- [Getting started as a miner](#getting-started-as-a-miner)
+  - [Hardware recommendations](#hardware-recommendations)
+  - [All about deals](#all-about-deals)
+- [Getting rewards](#getting-rewards)
+  - [WinningPoSt](#winningpost)
+  - [Uptime, slashing and penalties](#uptime-slashing-and-penalties)
+  - [Window PoSt checks](#window-post-checks)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
```suggestion
[[TOC]]
```

Keeping a manual TOC is a bit fragile because not all editors will have the doctoc plugin. In most cases the TOC will be redundant because of the sidebar nav but here you can use the magic `[[TOC]]` token to auto generate this at build time, keeping everything in sync 👌

Reply:
Oh nice! did not know that